### PR TITLE
Feature/defra id login

### PR DESCRIPTION
**Pull request by alex742**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ffc-ahwr-dashboard",
-  "version": "0.2.0",
+  "version": "0.3.0",
```
Comment (capJuan) [inline package.json:3]: What is the policy on the version ,  we usually change the last digit for small changes

File: package.json [hunk 2]
```diff
@@ -45,16 +45,15 @@
     "jsonwebtoken": "^9.0.0",
     "jwk-to-pem": "^2.0.5",
     "mssql": "^10.0.1",
-    "notifications-node-client": "^7.0.3",
+    "notifications-node-client": "^7.0.6",
```
Comment (capJuan) [inline package.json:48]: if upgrading the package does not resolve the Snyk issue ,  should we keep the same version as there is no benefit on upgrade the package

File: package.json [hunk 3]
```diff
@@ -45,16 +45,15 @@
...
     "nunjucks": "^3.2.3",
     "pg-promise": "^11.5.4",
     "uuid": "^8.3.2",
     "webdriverio": "^8.16.12"
   },
   "overrides": {
     "notifications-node-client": {
-      "jsonwebtoken": "9.0.0",
-      "axios": "1.6.0"
+      "jsonwebtoken": "9.0.0"
```
Comment (capJuan) [inline package.json:56]: same as above

File: package.json (hunk 2 above)
Comment (alex742) [reply package.json:48]: This resolves the axios issue and removes the need for the axios override that I also removed

File: package.json (hunk 1 above)
Comment (alex742) [reply package.json:3]: True, I considered adding authentication and logging in to be a larger change than usual. We don't have an agreed upon set of rules for increasing version numbers. Happy to change it if you want?

**Review by capJuan — APPROVED** [review]

**Pull request merged**